mods/species/utility_frames/species.dm

```diff
@@ -62,7 +63,11 @@
 	)
 	has_organ = list(
 		BP_POSIBRAIN = /obj/item/organ/internal/posibrain,
-		BP_EYES = /obj/item/organ/internal/eyes/robot
+		BP_EYES      = /obj/item/organ/internal/eyes/robot/utility_frame, //Eyes got a different icon file 
+		BP_CELL      = /obj/item/organ/internal/cell,
+	)
+	override_limb_types = list(
+		BP_HEAD = /obj/item/organ/external/head/utility_frame, //Needed for glowing eyes shennanigans
```

Comment:
Comments after list entries will fail the indentation check. Also trailing comma here and above.

Reply:
Oh okay.